onprc_ehr/resources/web/onprc_ehr/panel/SnapshotPanel.js

```diff
@@ -284,9 +284,8 @@ Ext4.define('onprc_ehr.panel.SnapshotPanel', {
 
 
         //Modified: 2-14-2017  R.Blasa  Make animal group name as hyperlink
-        var text = values.length ? values.join('<br>') : 'None';
-        var url =  '<a href="' + LABKEY.ActionURL.buildURL('query', 'executeQuery', null, {schemaName: 'ehr', 'query.queryName': 'animal_groups',
-                  'query.name~eq': values.length ? values.join(';') : 'None' }) + '" target="_blank">' + text  + '</a>';
+        var url =  values.length ? '<a href="' + LABKEY.ActionURL.buildURL('query', 'executeQuery', null, {schemaName: 'ehr', 'query.queryName': 'animal_groups',
```

Comment:
Missing URL and HTML encoding.

Reply:
Good catch. I added the HTML encoding. ActionURL.buildURL should automatically URL encode the parameters if that is what your concern is there.